titty

```diff
@@ -44,10 +44,10 @@ tdisp(){
 	total=0
 	while read -r NAME START STOP
 	do
-		if [ "$1" = 'all' ] || [ "$(date --date=@"$START" +"$FMTSTRING")" -eq "$(date +"$FMTSTRING")" ]
+		if [ "$1" = 'all' ] || [ "$(date --date=@$START +$FMTSTRING)" -eq "$(date +$FMTSTRING)" ]
 		then
-			TIME=$(("$STOP" - "$START"))
-			total=$(("$total" + "$TIME"))
+			TIME=$(($STOP - $START))
```

Comment:
This is good, quotes are not needed here since the variables contain number. That being said there is no harm in keeping them too